### ISSUE TITLE
New Category: <Drakewatcher Manuscripts>

### ISSUE THREAD

**Issue by rainkius** [Zottelchens-WoW-Addons/AdiBags-Dragonflight#3]
### Category Name

Drakewatcher Manuscripts

### In-Game Color

#33937F

### In-Game Description

Drakewatcher Manuscripts

### README.md Description

Drakewatcher Manuscripts for learning new customizations for your Dragonriding mounts

### Item IDs

```Shell
196961
196986
196991
197013
196987
197012
197019
196996
196965
197000
196981
196979
197015
196973
196982
196969
197001
196984
197022
197018
197002
196974
196964
196966
196967
197011
196975
196976
196992
196990
196998
196985
197005
197017
196983
197023
197016
197008
196972
197006
196988
197010
197009
196994
196963
196962
196993
196978
197020
196970
196971
197003
197021
196995
197004
196977
196997
196968
196999
196980
196989
197014
197007
197099
197117
197142
197153
197156
197145
197118
197101
197149
197125
197100
197128
197116
201792
197098
197106
197155
197090
197094
197095
197127
197143
197131
197122
197147
197119
197126
197152
197102
197139
197103
197111
197114
197120
197110
197144
197141
197091
197093
197112
197129
197140
197132
197150
197109
197134
197151
197097
197133
197105
197108
197154
197137
197130
197096
197138
197124
197113
197121
197104
197136
197123
197115
197135
197107
197148
197146
197357
197401
197348
197392
197368
197390
197377
197353
197391
197369
197403
197375
197380
197358
197366
197376
201790
197365
197388
197404
197408
197346
197351
197349
197381
197367
197371
192523
197389
197356
197359
197395
197383
197397
197373
197354
197385
197379
197363
197405
197394
197372
197399
197370
192111
197400
197364
197347
197350
197396
197398
197402
197361
197386
197407
197360
197362
197406
197352
197378
197374
197355
197384
197387
197382
197393
199074
197588
197620
197597
197611
197612
197577
197613
197624
197602
197605
197603
197583
197626
197621
197587
197630
197593
197625
197584
197595
197627
197622
197580
197598
197608
197591
197617
197600
197619
197596
197589
197618
197585
197604
197628
197635
197599
197614
197636
197578
197581
197594
197590
197586
197629
197623
197592
197607
197579
197606
197615
197601
197582
197609
197616
197634
197610
```

**Comment by rainkius** [comment]
I know there are a lot of item IDs here... They are from Wowhead, sorted alphabetically: Items > Misc > Other: Added in Expansion Dragonflight + Column Filter: Drakewatcher

Thank you for your filters ^_^

**Comment by Zottelchen** [comment]
Thank you so much for sharing this list! It is implemented and will appear in the version 1.1.6 soon! :)
